### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by justinfagnani**
We need more detailed instructions in the template....

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+
+This PR is a:
+- [ ] Bug Fix
+- [ ] Feature
+
+Checklist:
+- [ ] Issue 
+- [ ] Tests
+- [ ] Update CHANGELOG.md
+- [ ] CLA
```
Comment (kevinpschaaf) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] Submitter has signed the CLA
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+
+<!-- Description -->
+<!-- Please include "Fixes #X" so GitHub can track PRs against issues. -->
+
+This PR is a:
+- [ ] Bug Fix
+- [ ] Feature
+
+Checklist:
+- [ ] Issue
```
Comment (kevinpschaaf) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] Description includes issue this PR fixes
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+<!-- Description -->
+<!-- Please include "Fixes #X" so GitHub can track PRs against issues. -->
+
+This PR is a:
+- [ ] Bug Fix
+- [ ] Feature
+
+Checklist:
+- [ ] Issue 
+- [ ] Tests
```
Comment (kevinpschaaf) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] PR includes tests for this change
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,3 +1,21 @@
...
+- [ ] Update CHANGELOG.md
```
Comment (kevinpschaaf) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] PR includes update to CHANGELOG.md
```

**Comment by justinfagnani** [comment]
Thanks, those are better :)

**Comment by justinfagnani** [comment]
Obsolete

**Pull request closed without merging**